### DISCUSSION TITLE
Add Discord Channels?

### DISCUSSION THREAD

**Discussion by ewainy** [FrancescoXX/free-Web3-resources#53] [Ideas]
@JayrajSingh65 raised a great PR with Youtube links and Discord channels

**What do we all think for adding discord channels to the repo?**

I personally like the idea based on the way that @JayrajSingh65 has added these links

I like that the link takes you to discord street and the individual server, giving you the option to read about the discord server before joining instead of a direct invite link

Please share your thoughts 😊

**Comment by FrancescoXX** [comment]
We add the discord usually as communities, if they are worth it of course
